recipes/blas/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set blas_pkg = "openblas" %}
+
+{% set version = "1.0.0" %}
+{% set build_number = 0 %}
+
+{% set version_post = (".post" + build_number|string) if build_number|int != 0 else "" %}
+{% set version_full = version + version_post %}
```

Comment:
This is my attempt to still have build numbers, but honestly it feels like a waste. I'd be happy to drop this stuff.